bench/hamming/complex.fpcore

```diff
@@ -0,0 +1,43 @@
+; -*- mode: scheme -*-
+
+#;(FPCore (xre xim)
```

Comment:
Best to add a comment explaining why they're commented out, in case later we forget. Something like, "TODO: exp function unimplemented."